metric-example/query-by-example.md

```diff
@@ -183,7 +183,7 @@ To check for available aggregation techniques, obtain the array in `/metrics/<m
 | `count` | Determines the value count in the time slot, not counting `null` | Response time, Action duration |
 | `percentile(N)` | Estimates the n-th percentile, where N in range [0,100) is mapped to p in range [0,1) | Response time, Action duration |
 
-Percentile aggregations are available for many response time-based metrics. The count is often useful to determine the reliability of quantile estimations. Generally, more samples allow for a more exact estimation or more exact averages.
+Percentile aggregations are available for many response- time based metrics. The count is often useful to determine the reliability of quantile estimations. Generally, more samples allow for a more exact estimation or more exact averages.

```

Comment:
```suggestion
Percentile aggregations are available for many response time based metrics. The count is often useful to determine the reliability of quantile estimations. Generally, more samples allow for a more exact estimation or more exact averages.

```
I'm actually not sure what the best way is to write "response time based", but it's no right the way it is now 😄

Reply:
```suggestion
Percentile aggregations are available for many response-time-based metrics. The count is often useful to determine the reliability of quantile estimations. Generally, more samples allow for a more exact estimation or more exact averages.

```
I'd go this way.